docs/embedding/guest-embedding.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Guest embeds
+summary: Embed questions, dashboards, and documents without requiring SSO.
+---
+
+# Guest embeds
+
+{% include shared/in-page-promo-embedding-workshop.html %}
+
+Guest embeds are a way to embed basic Metabase components in your app without requiring you to create an account for each person in your Metabase. But not logging people in to your Metabase has some major tradeoffs: see [limitations](#guest-embed-limitations).
```

Comment:
for each person using charts/dashboard?.. "for each person" is weird without a qualifier